### ISSUE TITLE
Configured Rutracker.ru then all active indexers disappeared from console

### ISSUE THREAD

**Issue by duggerz** [Jackett/Jackett#15394]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
2024-06-09 14:51:33	Info	
Using HTTP Client: HttpWebClient2
2024-06-09 14:51:33	Info	
Using FlareSolverr: No
2024-06-09 14:51:33	Info	
Using proxy: Disabled
2024-06-09 14:51:33	Info	
App config/log directory: /home/ian/.config/Jackett
2024-06-09 14:51:33	Info	
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
2024-06-09 14:51:33	Info	
Running in Docker: No
2024-06-09 14:51:33	Info	
File /etc/issue: Ubuntu 22.04.4 LTS \n \l
2024-06-09 14:51:33	Info	
Jackett variant: CoreLinuxAmdx64
2024-06-09 14:51:33	Info	
OS version: Unix 6.5.0.35 (64bit OS) (64bit process)
2024-06-09 14:51:33	Info	
Environment version: 8.0.6 (/opt/Jackett/)
2024-06-09 14:51:32	Info	
Starting Jackett v0.22.89
```

### Description

I added Rutracker.ru to the list of configured trackers and all have disappeared from the console though they are still working to resolve requests

### Logged Error Messages

```text
None found
```

**Comment by duggerz** [comment]
Previously I added Rutracker.ru and saw an error message in Russian.

**Comment by duggerz** [comment]
I can send the portion of the log that has all the indexers listed but I can't reproduce that action unless I wipe the config completely and start again, even then I don't know if it will reproduce.

Regards,
Ian
----------------------------------------

9 Jun 2024 16:32:28 Bogdan ***@***.***>:

**Comment by duggerz** [comment]
I did notice that there is a total of only 14 indexers now in the dashboard, and I'm surmising that only those are being retrieved from the config now.

Regards,
Ian
----------------------------------------

9 Jun 2024 16:51:57 Ian Dugdale ***@***.***>:

**Comment by ilike2burnthing** [comment]
A screenshot may also be useful, as I'm not 100% sure what you're talking about. Note that you can't send files by email, you'll have to do it on GitHub directly.

**Comment by garfield69** [comment]
I expect that this is simply a case of erroneous typing caught in the dashboard searchbox which returned no matches, or a dashboard filter that found no hits.

**Comment by duggerz** [comment]
Yes it was some text in the search box so the console was displaying 0 of 14.  Apologies, the problem was all me.
